lib/player.js

```diff
@@ -3422,8 +3422,8 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         const segmentTime = reference.endTime - reference.startTime;
         const thumbnailPosition =
             Math.floor(thumbnailTime * totalImages / segmentTime);
-        positionX = (thumbnailPosition % columns) / columns * fullImageWidth;
-        positionY = (thumbnailPosition % rows) / rows * fullImageHeight;
+        positionX = (thumbnailPosition % columns) * width;
+        positionY = Math.floor(thumbnailPosition / columns) * height;
```

Comment:
Should be: `Math.floor(thumbnailPosition / rows) * height;`

(columns vs rows)

Reply:
@avelad I've checked the logic again and the use of columns is correct. Consider the following as per the unit test.
Grid of 2x3 100x150px:
```
[0][1]
[2][3]
[4][5]
```

Requesting the position of thumbnail 5 using your suggested edit would result in the following:
```
Math.floor(thumbnailPosition / rows) * height
Math.floor( 5 / 3) * 50
1 * 50
yPosition = 50
```

The correct position for thumbnail 5 should be:
```
x= 50
y= 100
```